src/main/java/appeng/core/sync/packets/PacketProgressBar.java

```diff
@@ -20,27 +20,27 @@
 
 
 import io.netty.buffer.ByteBuf;
-import io.netty.buffer.Unpooled;
 
-import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.client.Minecraft;
 import net.minecraft.inventory.Container;
 
+import cpw.mods.fml.common.network.simpleimpl.MessageContext;
+import cpw.mods.fml.relauncher.Side;
+
 import appeng.container.AEBaseContainer;
 import appeng.core.sync.AppEngPacket;
-import appeng.core.sync.network.INetworkInfo;
+import appeng.core.sync.AppEngPacketHandler;
 
 
-public class PacketProgressBar extends AppEngPacket
+public class PacketProgressBar implements AppEngPacket, AppEngPacketHandler<PacketProgressBar, AppEngPacket>
 {
 
-	final short id;
-	final long value;
+	private short id;
+	private long value;
 
 	// automatic.
-	public PacketProgressBar( ByteBuf stream )
+	public PacketProgressBar()
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1186)